README.md

```diff
@@ -1,4 +1,7 @@
 # OCaml-migrate-parsetree
+[Deprecated] This package is deprecated. Please, use
```

Comment:
Nitpick: I think the deprecation warning should be more visible. I don't think `[...]` is a markdown syntax for emphasizing a word. This [post](https://medium.com/maintainer-io/how-to-deprecate-a-repository-on-github-8f0ceb9155e) suggests changing the title, but maybe using bold face and caps lock for `deprecated` is enough?

```suggestion

**DEPRECATED** This package is deprecated. Please, use
```